sources/subs/Cache.class.php

```diff
@@ -57,6 +57,12 @@ class Cache
 	protected $_accelerator = null;
 
 	/**
+	 * Cached keys
+	 * @var string[]
+	 */
+	protected $_cached_keys = array();
```

Comment:
i would define this as static,
otherwise you could get issue when the same time this logic run twice.

Some addition checks in construct etc. would be than needed too.